### PR TITLE
ref(ui): converted suggestedOwners to ts

### PR DESCRIPTION
**Pull request by priscilawebdev**
Converted SuggestedOwners file to typescript

### CODE REVIEW

File: src/sentry/static/sentry/app/components/group/suggestedOwners/ownershipRules.tsx
Comment (markstory) [inline src/sentry/static/sentry/app/components/group/suggestedOwners/ownershipRules.tsx:47]: Should this become `IconQuestion`? We use that icon in other situations where we are giving contextual help.

File: src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx
Comment (markstory) [inline src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx:5]: Should we use an actual heading element here?

File: src/sentry/static/sentry/less/sentry-hovercard.less
Comment (markstory) [inline src/sentry/static/sentry/less/sentry-hovercard.less:86]: :clap:

File: src/sentry/static/sentry/app/components/group/suggestedOwners/ownershipRules.tsx
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/ownershipRules.tsx:47]: sure thing!

File: src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx]: you are right, headings are important, but I believe that we should review the spaces that accompany our h* elements ... otherwise, I have to use` !important` everywhere .. and this is not good practice. 

I have updated the code 😉

File: src/sentry/static/sentry/app/components/avatar/actorAvatar.tsx
Comment (matejminar) [inline src/sentry/static/sentry/app/components/avatar/actorAvatar.tsx:33]: Should you use default props instead?

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/avatar/actorAvatar.tsx:33]: sometimes default Props doesn't play nice with typescript... but I believe that in this case it will be fine... I will update 😉

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (matejminar) [inline src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:61]: Should we make async component out of this while we are refactoring?

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (matejminar) [inline src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:147]: I find owners.some(...) really handy for this kind of use-case.

File: src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx
Comment (markstory) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx]: Would having a module of styled components that provides some headings with the margins stripped off help? I agree that having to apply important all over the place is not great.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:61]: Please why do you think that it is not a good idea? it's a request... it must be async

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:147]: Yes I agree with you, but in this case I am preferring to leave findIndex here since I am using it to update the owners array...otherwise I would have to map the owners and add a condition in the loop in order to update the correct one.

Comment (matejminar) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:147]: Oh yeah, didn't notice the update 👍

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:90]: Let's not mix await with then/catch

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx]: thank you for the tip Billy... Code is updated 😉

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx]: 👍

File: src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx
Comment (billyvg) [reply src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedOwners.tsx:61]: I would suggest leaving it as there's some different fetching logic here and this needs to be refactored a bit so we're not making duplicate requests to `/committers/`.

**Review by billyvg — APPROVED** [review]

**Pull request merged**
